inputs/values/ref_panel_1kg.json

```diff
@@ -0,0 +1,3032 @@
+{
+    "PE_files": [
```

Comment:
Ah this is because they are capitalized. I could go through and change the names to lower-case but this will be a bit of trouble. We can try to address this in #16